src/qmozview_p.h

```diff
@@ -174,6 +175,7 @@ public Q_SLOTS:
     mozilla::embedlite::EmbedLiteView *mView;
     bool mViewInitialized;
     unsigned mParentID;
+    uintptr_t mParentBrowsingContext;
```

Comment:
Not an big issue but `mParentBrowsingContext` should be introduced in a previous commit as it has been started used already there.

Reply:
Will check and amend, this commit could even be squashed to the previous. I did do "git add -p" picking but apparently failed there. :)

Reply:
Squashed and left two change log entries within the commit message.